src/defines.h

```diff
@@ -18,11 +18,12 @@
  * http://www.gnu.org/copyleft/gpl.html
  */
 
-#ifndef DEFINES_H
-#define DEFINES_H
+#ifndef SRC_DEFINES_H
+#define SRC_DEFINES_H
```

Comment:
Should this not be called `SRC_DEFINES_H_` or does the last underscore not matter?